### PR TITLE
Try to make bundle size smaller

### PR DESCRIPTION
**Pull request by Utwo**
### What changed?
Exported everything with a specific named export. 
Add module:false in babel config.
Add commonjs plugin in rollup config.

I still don't know if this is enough for the consumer app to tree-shake this or not. Waiting for your help!

### Actions (optional)
 - [x] Increase the version of the package if you need to release it after the merge
 - [ ] If you added a new component, please make sure to export it in `../src/index.js`

### CODE REVIEW

**Comment by Utwo** [comment]
I will add here some articles for guidance. 
https://risanb.com/code/bundling-your-javascript-library-with-rollup/#three-shaking-gotcha
https://stackoverflow.com/questions/59900538/shared-component-library-best-practices

**Comment by Utwo** [comment]
I added a config to rollup in which I export in multiple bundles by hand. It's still not using tree-shaking feature but it's still a plus.

Before:
old

After:
new

**Comment by Utwo** [comment]
I think we still need to find a way to use the tree-shaking capabilities

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "@code4ro/taskforce-fe-components",
   "version": "1.0.24",
```
Comment (tudorgergely) [inline package.json:3]: we should bump the version

**Pull request merged**